pingcap/tidb
# store/copr: invalidate stale regions for Mpp query.

### What problem does this PR solve?

Problem Summary:
Right now, we won't update the stale regions during executing Mpp queries. Then we added the try regions field in MppDispatchTask reponse. In this pr, we try to invalidate the stale regions in the mppr response.

### What is changed and how it works?

### Related changes
https://github.com/pingcap/kvproto/pull/751

### Check List 

Tests 

- Manual test (add detailed scripts or steps below)
  - Insert data continously
  - observe that tiflash is retrying regions. In my case, the log is like `["DAGQueryBlockInterpreter: Start to retry 1 regions (10936,)"]`
  - observe log in tidb, find `["invalid region because tiflash detected stale region"] ["region id"="{ region id: 10936, ver: 116, confVer: 2 }"]` proving this function works.
 

### Release note 

- store/copr: invalidate stale regions for Mpp query.
